src/sentry/integrations/vsts/search.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from rest_framework.response import Response
+
+from sentry.models import Integration
+
+
+class VstsSearchEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationPermission, )
+
+    def get(self, request, organization, integration_id):
+        try:
+            integration = Integration.objects.get(
+                organizations=organization,
+                id=integration_id,
+                provider='vsts',
+            )
+        except Integration.DoesNotExist:
+            return Response(status=404)
+
+        field = request.GET.get('field')
+        query = request.GET.get('query')
+        if field is None:
+            return Response({'detail': 'field is a required parameter'}, status=400)
+        if not query:
+            return Response({'detail': 'query is a required parameter'}, status=400)
+
+        installation = integration.get_installation(organization.id)
+
+        if field == 'externalIssue':
+            if not query:
+                return Response
```

Comment:
you probably want to actually instantiate this?

Reply:
Don't you already check `if not query` above?